apps/aecore/test/aecore_sync_SUITE.erl

```diff
@@ -466,8 +518,8 @@ unsubscribe(N, Event) ->
 call_proxy(N, Req) ->
     call_proxy(N, Req, ?PROXY_CALL_RETRIES, 3000).
 
-call_proxy(N, Req, Timeout) ->
-    call_proxy(N, Req, ?PROXY_CALL_RETRIES, Timeout).
+%% call_proxy(N, Req, Timeout) ->
+%%     call_proxy(N, Req, ?PROXY_CALL_RETRIES, Timeout).
```

Comment:
Just remove?

Reply:
I tend to want to comment out some functions if I think it likely that they may be used in future test cases.